fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowReplicaDistributionCommand.java

```diff
@@ -0,0 +1,108 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.nereids.trees.plans.commands;
+
+import org.apache.doris.analysis.PartitionNames;
+import org.apache.doris.analysis.RedirectStatus;
+import org.apache.doris.catalog.Column;
+import org.apache.doris.catalog.Env;
+import org.apache.doris.catalog.MetadataViewer;
+import org.apache.doris.catalog.ScalarType;
+import org.apache.doris.common.AnalysisException;
+import org.apache.doris.common.DdlException;
+import org.apache.doris.common.ErrorCode;
+import org.apache.doris.common.ErrorReport;
+import org.apache.doris.common.util.Util;
+import org.apache.doris.mysql.privilege.PrivPredicate;
+import org.apache.doris.nereids.trees.plans.PlanType;
+import org.apache.doris.nereids.trees.plans.commands.info.TableRefInfo;
+import org.apache.doris.nereids.trees.plans.visitor.PlanVisitor;
+import org.apache.doris.qe.ConnectContext;
+import org.apache.doris.qe.ShowResultSet;
+import org.apache.doris.qe.ShowResultSetMetaData;
+import org.apache.doris.qe.StmtExecutor;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
+/**
+ * show replica distribution command
+ */
+public class ShowReplicaDistributionCommand extends ShowCommand {
```

Comment:
please help update the two functions in Command.java (add ConnectContext as param)
```
public void verifyCommandSupported(ConnectContext ctx) throws DdlException {
    // check command has been supported in cloud mode
    if (Config.isCloudMode()) {
        checkSupportedInCloudMode(ctx);
    }
}
```
`
protected void checkSupportedInCloudMode(ConnectContext ctx) throws DdlException {}`

and then change the caller in StmtExecutor.java
`((Command) logicalPlan).verifyCommandSupported(context);`

at last, override checkSupportedInCloudMode method as bellow
```
protected void checkSupportedInCloudMode(ConnectContext ctx) throws DdlException {
    if (!ctx.getCurrentUserIdentity().getUser().equals(Auth.ROOT_USER)) {
        LOG.info("stmt={}, not supported in cloud mode", stmt.toString());
        throw new AnalysisException("Unsupported operation");
    }
}
```